mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -633,6 +651,25 @@ impl Nexus {
         Ok(())
     }
 
+    /// set ANA state of the NVMe subsystem
+    pub async fn set_ana_state(
+        &self,
+        ana_state: NvmeAnaState,
+    ) -> Result<(), Error> {
+        if let Some(Protocol::Nvmf) = self.shared() {
+            if let Some(subsystem) = NvmfSubsystem::nqn_lookup(&self.name) {
+                subsystem.pause().await.unwrap();
```

Comment:
As guilty as others, but as we move along, we should really stop doing the unwrapping and handle the errors properly. If we can't handle the errors properly, then at least let us add .exect("huh?") (much better this way, but you get the point =))

Reply:
`.expect("huh?")` will impact our error messages. It's better to return the error for a backtrace if we can.

Reply:
As there is now something that converts from `subsys::NvmfError` to this `Error` I've opted to return the error via `?`.